code/src/components/Header.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import moment from "moment";
```

Comment:
Since you've installed the library 'react-moment', this import is actually importing the dependency 'moment' _from_ the 'react-moment' dependency. This can lead to unexpected things if you're not careful. I'd `npm uninstall react-moment` and replace it with the actual dependency; `npm install moment`, so the actual dependency is clear.

This is me being super picky, but it's good to be aware that importing dependencies of your dependencies is an npm-specific "feature" which is not super reliable. There are other package managers such as [yarn](https://yarnpkg.com/lang/en/) (my favourite! :) ) and [pnpm](https://pnpm.js.org/) where this import would actually fail since it's not a top-level dependency.